vpc/eu-central-1/main.tf

```diff
@@ -16,29 +17,32 @@ locals {
 
   vpc_tags = {
     Name                                                      = "k8s.eu-central-1a.mdn.mozit.cloud"
+    Region                                                    = "eu-central-1"
     KubernetesCluster                                         = "k8s.eu-central-1a.mdn.mozit.cloud"
-    "kubernetes.io/cluster/k8s.eu-central-1a.mdn.mozit.cloud" = "shared"
+    "kubernetes.io/cluster/k8s.eu-central-1a.mdn.mozit.cloud" = "owned"
   }
 
   subnet_tags = {
-    "kubernetes.io/cluster/k8s.eu-central-1a.mdn.mozit.cloud" = "shared"
+    Region                                                    = "eu-central-1"
+    "kubernetes.io/cluster/k8s.eu-central-1a.mdn.mozit.cloud" = "owned"
```

Comment:
Does this tag has any effect in the AWS k8s magic?

Reply:
No if you look at the docs it says it requires "owned" and not "shared"